modules/tracker/mbt/src/vpMbtXmlGenericParser.cpp

```diff
@@ -1447,6 +1447,7 @@ class vpMbtXmlGenericParser::Impl
 
 vpMbtXmlGenericParser::vpMbtXmlGenericParser(int type) :  m_impl(new Impl(type))
 {
+  std::setlocale(LC_ALL, "en_US.UTF-8");
```

Comment:
Needed when calling from Java. Java defines somehow the locale to current host and PugiXML uses it to parse decimal according to the locale (issue with comma / point decimal character).